tests/testthat/test-loadData.R

```diff
@@ -0,0 +1,34 @@
+#Test files for function loadData in models.R
+test_that('loadData report error if the includeVaccines contains vaccines that are not prescribed',{
+  par = Parameters$new()
+  expect_error(loadData(par = par, candidateFile = 'extdata/vaccinesSummary.csv',includeVaccines = 1))
```

Comment:
Unfortunately these have to be modified a bit because we want to test both with `system.file()` argument and as a file path (see comments elsewhere).